static/css/style.css

```diff
@@ -1,74 +1,130 @@
 :root {
-    /* P = Primary | S = Secondary | T = Tertiary */
-    --p-orange: #E84610;
-    --p-blue: #009FE3;
-    --p-grey: #4A4A4F;
-    --p-navy: #445261;
-    --p-red: #D63649;
-    --s-grey: #E6ECF0;
-    --s-yellow: #EFB920;
-    --s-pink: #FF5A60;
-    --s-teal: #23BBBB;
-    --t-grey: #E0E7FF;
-    --white: #FFFFFF;
-    --black: #000000;
-    --bg-grey: #F5F5F5;
-}
-
-html, body {
-    height: 100%;
+  /* P = Primary | S = Secondary | T = Tertiary */

```

Comment:
@JimLynx as discussed, can you investigate this (probably an extension you have), and revert these back please?